services/bonita-time-tracker/src/test/java/org/bonitasoft/engine/tracking/RecordAssert.java

```diff
@@ -0,0 +1,121 @@
+package org.bonitasoft.engine.tracking;
```

Comment:
i didn't modify the generated file. 
this code should be re-generated is class under assertion is modified